src/Trixi.jl

```diff
@@ -289,6 +289,8 @@ export PlotData1D, PlotData2D, ScalarPlotData2D, getmesh, adapt_to_mesh_level!,
        adapt_to_mesh_level,
        iplot, iplot!
 
+export GmshFile, AbaqusFile
+
```

Comment:
These do not seem to be used anywhere in the examples. Do we really need to export them?

Generally, I have no issue with adding new `export`s, I'd just like to avoid spilling internals unnecessarily to the public API

Reply:
Fair enough. I remove them.